server/athenian/api/internal/miners/github/deployment.py

```diff
@@ -242,14 +244,13 @@ async def mine_deployments(
             rdb,
             cache,
         ),
-        name="_fetch_precomputed_deployed_releases(%d)" % len(notifications),
```

Comment:
The task has a generic name by default. I try to give meaningful names to the tasks to simplify the debugging.